MCprep_addon/spawner/mcmodel.py

```diff
@@ -378,7 +384,9 @@ def add_model(
 				face.material_index = materials.index(face_mat)
 
 	# Quick way to clean the model, hopefully it doesn't cause any UV issues
-	bmesh.ops.remove_doubles(bm, verts=bm.verts, dist=0.01)
+	# Ignore model has overlay geometry, causing issue
+	if not textures.get("overlay"):
+		bmesh.ops.remove_doubles(bm, verts=bm.verts, dist=0.01)
```

Comment:
Per above suggestion, to ensure enough margin

```suggestion
		bmesh.ops.remove_doubles(bm, verts=bm.verts, dist=0.005)
```